packages/dds/map/src/test/mocha/map.fuzz.spec.ts

```diff
@@ -25,7 +25,7 @@ interface Clear {
 interface SetKey {
 	type: "setKey";
 	key: string;
-	value: Jsonable;
+	value: JsonableType;
```

Comment:
It could be `Jsonable<unknown>` or `JsonableType`, which are now the same. `Jsonable<>` is meant to be a filter for the Jsonable portions of given type param and there is nothing to filter here; so just `JsonableType` seemed most appropriate.

Reply:
In places that didn't have a use for `Jsonable` like this file, I would fully switch to `JsonableType`. If `Jsonable<>` was otherwise in use I used `Jsonable<unknown>` for symmetry/consistency.

Reply:
I am not in favor of the `Jsonable` rename. 1) It isn't really an assert. It produces a new type that might be useful. I think an assert would come from checking that `T extends JsonableType`. 2) Rename is more breaking than this change; good existing uses should be fine.
I am open to alternate names for `JsonableType` and `JsonableTypeWith<>`. tree2's types are very similar. Big difference looks like handling of `undefined`. (Would like the datastore-definitions to have better handling for the deserialized typing which does not support `undefined`.) I would not want to use `JsonComaptible` naming unless we reconcile the types. Too confusing otherwise.
I could see changing `Jsonable<>` to `JsonableSubset<>` or `JsonSubset<>` if we think a bigger breaking change is worth it. Then, assuming `Type` is not preferred in type named, `JsonableType` could just be `Jsonable` (type for a value that can be effectively* serialized directly to Json excluding some caveats (* - doesn't mean it would be deserialized exactly (the `undefined` issue))).

Reply:
I see also that tree2 doesn't do generate type compat testing. Is there "manual" testing for compat?

Reply:
Tree2 has not yet been released / used in production.  (It's our big 2.0 feature.)

If Jsonable<unknown> and JsonableType are interchangeable, I would think it's easier to explain that we have a single type that can do both.

(It's what we tried to do with the original Jsonable<any>, but unfortunately didn't get the result we wanted...  but left it in as documentation.)

Reply:
`Jsonable` and `JsonableType` are not interchangeable. `JsonableType` is an expression of what of `any` can be serialized to json. `Jsonable` is a utility to get the json serializable portions of a type. A qualified `Jsonable<Foo>` is a specific type. `Jsonable<any | unknown>` maps to `JsonableType`, but no other case should. We can use `Jsonable<unknown>` broadly, but I think it is nicer to just use the thing that it will be (`JsonableType`) rather than having people need to understand that `Jsonable<unknown>` is a special case.
In this file, we are looking at test code (internal) and `JsonableType` as the most clear type makes sense to me.

Reply:
FWIW I think the most exposure will really be via `Serializable<>`. I did not create a canonical base type for that one as I wasn't seeing so much use. In general, I think we hope to avoid `Serializable<unknown>`.

Reply:
Well, I looked for uses and this was really the only case. I have plenty more in my prototype, but those may yet become Serializable. So, I'll go for the path of `JsonableType` being an implementation detail for now.

Reply:
In my prototype the most common use of `JsonableType` is for a generic constraint.
 has:
```typescript
function readJson<T extends Jsonable<unknown>>(filepath: string): Jsonable<T> {
```
but I would instead do
```typescript
function readJson<T extends JsonableType>(filepath: string): Jsonable<T>
// or
function readJson<T extends JsonableType>(filepath: string): T
```
(I haven't explored if there is a meaningful difference in `Jsonable<T>` versus `T` when `T extends `JsonableType`, but I am imagining there is a use when there are distinct types for serializable versus deserialized cases.)

Reply:
`JsonableType` is no longer in this PR.
I will resolve this conversation, but feel free to unresolve to discuss naming changes that should happen now.